topo/node/node.go

```diff
@@ -512,8 +514,13 @@ var (
 func GetNextPort() uint32 {
 	muPort.Lock()
 	defer muPort.Unlock()
-	p := nextPort
-	nextPort++
+	ok := true
```

Comment:
why is this being added as part of this pull Request?

Reply:
I have reverted those changes. Those changes was made for another issue and it got accidentally pushed with the PR still open.